tests/Unit/ApparentHorizons/Test_StrahlkorperGr.cpp

```diff
@@ -129,11 +142,25 @@ void test_minkowski() {
   Approx custom_approx = Approx::custom().epsilon(1.e-12);
   CHECK_ITERABLE_CUSTOM_APPROX(
       get(residual), DataVector(get(residual).size(), 1.0), custom_approx);
+
```

Comment:
where is the test for extrinsic_curvature

Reply:
the minkowski ricci_scalar() test tests the extrinsic curvature, as only those terms are nonzero in that test.